src/components/Select.tsx

```diff
@@ -0,0 +1,75 @@
+import * as React from 'react';
+import { compose } from 'recompose';
+import { SelectProps } from 'rendition';
+import styled, { StyledFunction, withTheme } from 'styled-components';
+import asRendition from '../asRendition';
+import { radius } from '../theme';
+import { px } from '../utils';
+
+const Base = (styled.select as StyledFunction<
+	{ emphasized?: boolean } & React.HTMLProps<HTMLSelectElement>
+>)`
+  border-radius: ${px(radius)};
+  height: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
+  font-size: inherit;
+  border: 1px solid ${props => props.theme.colors.gray.main};
+  padding-top: 0px;
+  padding-bottom: 0px;
+  padding-left: 16px;
+  padding-right: ${props =>
+		px(props.emphasized ? props.theme.space[5] : props.theme.space[4])};
+  background-color: white;
+  -webkit-appearance: none;
+  -moz-appearance: none;
+  appearance: none;
+  width: 100%;
+  min-width: 90px;
+
+  &:hover {
+    box-shadow: 0 0 4px 1px rgba(0, 0, 0, 0.1);
+  }
+`;
+
+const Wrapper = (styled.span as StyledFunction<SelectProps>)`
```

Comment:
Could be `withProps<SelectProps>()(styled.span)` but it might be an overkill for this case.
PS: Even though I like it more since it also re-exposes the span prop typings.

Reply:
cool